src/Model/Notification.php

```diff
@@ -154,4 +157,160 @@ public static function getType(array $notification): string
 
 		return $type;
 	}
+
+	/**
+	 * Create a notification message for the given notification
+	 *
+	 * @param array $notification
+	 * @return array with the elements "causer", "notification", "plain" and "rich"
+	 */
+	public static function getMessage(array $notification)
+	{
+		$message = [];
+
+		if ($notification['type'] == Post\UserNotification::NOTIF_NONE) {
+			return $message;
+		}
+
+		if (empty($notification['target-uri-id'])) {
+			return $message;
+		}
+
+		$user = User::getById($notification['uid']);
+		if (empty($user)) {
+			Logger::info('User not found', ['application' => $notification['uid']]);
```

Comment:
```suggestion
			$this->logger->info('User not found', ['application' => $notification['uid']]);
```

Reply:
When called statically this leads to an error:
![Bildschirmfoto vom 2021-08-19 19-53-10](https://user-images.githubusercontent.com/844208/130119550-5c1d556e-8e16-4673-a9cb-2cf8ba54de34.png)

Reply:
Ah, never mind, you're right. That's because it shouldn't be called statically!